sdk/appconfiguration/app-configuration/README.md

```diff
@@ -25,53 +22,45 @@ Use the client library for App Configuration to:
 npm install @azure/app-configuration
 ```
 
-## Key concepts
-
-### Configuration Setting
-
-A Configuration Setting is the fundamental resource within a Configuration Store.
-In its simplest form, it is a key and a value. However, there are additional properties such as 
-the modifiable content type and tags fields that allows the value to be interpreted or associated 
-in different ways.
-
-The `label` property of a Configuration Setting provides a way to separate configuration settings 
-into different dimensions. These dimensions are user defined and can take any form. Some common 
-examples of dimensions to use for a label include regions, semantic versions, or environments. 
-Many applications have a required set of configuration keys that have varying values as the 
-application exists across different dimensions.
-
-For example, MaxRequests may be 100 in "NorthAmerica", and 200 in "WestEurope". By creating a 
-Configuration Setting named MaxRequests with a label of "NorthAmerica" and another, only with 
-a different value, in the "WestEurope" label, an application can seamlessly retrieve 
-Configuration Settings as it runs in these two dimensions.
-
 ## Examples
 
-#### nodejs - Authentication, client creation and listConfigurationSettings as an example written in TypeScript.
+#### nodejs - Creating and getting a setting in Javascript
 
 ##### Sample code
 
-```typescript
-import { AppConfigurationClient } from "@azure/app-configuration";
+```javascript
+const appConfig = require("@azure/app-configuration");
+
+const client = new appConfig.AppConfigurationClient("<App Configuration connection string goes here>");
```

Comment:
maybe a comment about how to do this better/different in production or not commit this to source control?

Reply:
For the simple readme sample I think this is more or less a standard thing, TBH.

The more full-featured samples do use dotenv which does lead to a nicer pattern, however.